### DISCUSSION TITLE
GDrive Documents

### DISCUSSION THREAD

**Discussion by ericchen59** [DiscipleTools/disciple-tools-mobile-app#567] [Show and tell]
I've got a [Google Drive folder of documents](https://drive.google.com/drive/folders/1zqxzbVzmRmQ-v9726GawfF1Bsq-K6tNP?usp=sharing) that help us along the way. 

You'll need to request access.

- Onboarding D.T.-wide contributors
- Problem Statement worksheet
- Problem Statements for Mobile and Web

**Comment by zdmc23** [comment]
Hi @ericchen59 !  Have we discussed yet how to transition these docs?  Thx!

**Reply by ericchen59** [reply]
Hey @zdmc23, we haven't yet.  The document in here that could most likely go _public_ is the onboarding document.

The problem statements are specific to the FaithTech x Disciple.Tools collaboration so far, that might live with @ahillbilly or Jordan B.

What do you think?
